lib/rule-helpers.js

```diff
@@ -0,0 +1,59 @@
+'use babel';
+
+const DOCUMENTATION_LIFETIME = 86400 * 1000; // 1 day TODO: Configurable?
```

Comment:
I'd say 1 day is sufficient. If the user wants to force the cache to update they can simply restart Atom. 🔥 the comment 😉.